srcpkgs/caffeine-ng/template

```diff
@@ -1,26 +1,19 @@
 # Template file for 'caffeine-ng'
 pkgname=caffeine-ng
-version=3.5.1
-revision=2
+version=4.0.2
+revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools_scm"
```

Comment:
needs scdoc for manpages

Reply:
Same as above, it's still there within the [source](https://codeberg.org/WhyNotHugo/caffeine-ng/src/tag/v4.0.2/share/man/man1) in this version. The next version won't even adhere to the `build_style=python3-module` by the looks of it.